_pytest/assertion/rewrite.py

```diff
@@ -35,6 +35,12 @@
 REWRITE_NEWLINES = sys.version_info[:2] != (2, 7) and sys.version_info < (3, 2)
 ASCII_IS_DEFAULT_ENCODING = sys.version_info[0] < 3
 
+if sys.version_info >= (3,5):
```

Comment:
I just wanted to say thanks to @Carreau and @takluyver, your work here is greatly appreciated! :smile: